website/versioned_docs/version-0.9.0/cli.md

```diff
@@ -0,0 +1,357 @@
+---
```

Comment:
Is this file an intended change ? It looks like it belongs to version 0.9.0 already?

Reply:
Added the same file to v0.9.0 so build doesn't break because it is a net new file to /docs/next